.github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml

```diff
@@ -121,7 +139,7 @@ runs:
           --set 'global.environment=${{ inputs.environment }}' \
           --timeout ${{ inputs.helm_timeout }} \
           --values '${{ inputs.helm_dir }}/values-${{ inputs.environment }}.yaml' \
-          ${{ inputs.helm_additional_args }} \
+          "${helm_additional_args}" \
```

Comment:
not sure you want this in double quotes, since you want each additional argument to be passed through separately (not sure it matters either though tbh)